doc/whats_new/v1.1.rst

```diff
@@ -234,17 +245,6 @@ Changelog
   :func:`sklearn.set_config`.
   :pr:`22856` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
-- |Enhancement| The error message is improved when importing
```

Comment:
I don't really understand why they appear here since they are not part of 1.1.0

Reply:
they must have been merged after I made the PR for the RC and not marked as "to backport"

Reply:
Make sense